compiler-rt/lib/sanitizer_common/sanitizer_asm.h

```diff
@@ -53,6 +55,29 @@
 # define ASM_TAIL_CALL tail
 #endif
 
+#if defined(__mips64) && __mips_isa_rev < 6
+#  define C_ASM_TAIL_CALL(tfunc, ifunc)       \
+    "lui $t8, %hi(%neg(%gp_rel(" tfunc        \
```

Comment:
not sure that you need these lines breaks before )))

Reply:
I think that we should ignore it.